sorc/build_rtma3d_wrfarw.sh

```diff
@@ -22,7 +22,26 @@ cd ${BASE}/rtma3d_wrfarw.fd/WRFV3.9
 ./clean -aa
 ./clean -a
 ./clean
+
+# 1. compiling with original dyn_em/solve_em.orig.F (as a backup)
+cp -p configure.wrf.useme configure.wrf
+cp -p dyn_em/ dyn_em/solve_em.orig.F dyn_em/solve_em.F
```

Comment:
This seems to be extraneous.

Reply:
@MatthewMorris-NOAA This is a bug. Thank you for pointing it out!
It was fixed in the latest commit. See the updated PR #79.